syntaxes/Dafny.tmLanguage

```diff
@@ -26,7 +26,7 @@
 				</dict>
 			</dict>
 			<key>end</key>
-			<string>:=|:\||;|\)|\}|(?=(?:class|codatatype|const|constructor|datatype|function|greatest|iterator|least|lemma|method|module|newtype|predicate|trait|type|var)\b)</string>
+			<string>:=|:\||;|\)|\}|(?=(?:class|codatatype|const|constructor|datatype|function|greatest|iterator|least|lemma|method|module|newtype|opaque|predicate|trait|type|var)\b)</string>
```

Comment:
If this is intended to be a full list of keywords, there are some missing. abstract, static, ghost are in the same category (modifiers) as opaque

Reply:
```suggestion
			<string>:=|:\||;|\)|\}|(?=(?:class|codatatype|const|constructor|datatype|function|greatest|iterator|least|lemma|method|module|newtype|opaque|predicate|trait|type|var|abstract|static|ghost)\b)</string>
```